lmfit/ui/ipy_fitter.py

```diff
@@ -116,12 +107,11 @@ def close(self):
         self.max_checkbox.close()
 
     def _repr_html_(self):
-        box = Box()
+        box = HBox()
         box.children = [self.value_text, self.vary_checkbox,
                         self.min_text, self.min_checkbox,
                         self.max_text, self.max_checkbox]
         display(box)
```

Comment:
I'm not following. IPython 2.x supports widgets. Ideally, NotebookFitter should work on both Python 2.7 and 3.x and IPython 2.x and 3.x. The widget API changed between IPython 2.x and 3.x. Later releases in the 2.x series support both 2-style and 3-style widgets, but early releases (e.g., 2.0) only support 2-style. That is why I imported things in two different ways.